wiki/markdown/step-by-step-subscription-notification.md

```diff
@@ -178,7 +178,7 @@ file as “failedNotificationDataBaseName” and collection name as
 for a certain duration before being deleted. This time can be configured in
```

Comment:
I noticed the properties failedNotificationCollectionName and failedNotificationDataBaseName are still mentioned in this section also. They need to be updated with the new names.

Reply:
Already updated